boards/risc-v/esp32c3/esp32c3-devkit-rust-1/src/Makefile

```diff
@@ -41,6 +41,18 @@ ifeq ($(CONFIG_BOARDCTL_IOCTL),y)
 CSRCS += esp32c3_ioctl.c
 endif
 
+ifeq ($(CONFIG_ESP32C3_SPI),y)
+CSRCS += esp32c3_board_spi.c
+endif
+
+ifeq ($(CONFIG_SENSORS_APDS9960),y)
+  CSRCS += esp32c3_apds9960.c
```

Comment:
Some places have 2 spaces and other not. Please make it consistent

Reply:
Done, thank you!